declare ahead of time so it's not implicit which is invalid in c23
diff --git a/socks.c b/socks.c
index 7c7448c..ce2533d 100644
--- a/socks.c
+++ b/socks.c
@@ -53,6 +54,8 @@
 #define SOCKS_DOMAIN	3
 #define SOCKS_IPV6	4
 
+int b64_ntop(const uint8_t *, size_t, char *, size_t);
+
 int	remote_connect(const char *, const char *, struct addrinfo, char *);
 int	socks_connect(const char *, const char *, struct addrinfo,
 	    const char *, const char *, struct addrinfo, int,
